itl/command-plugins-manubulon.conf

```diff
@@ -149,15 +149,15 @@ object CheckCommand "snmp-storage" {
 }
 
 
-/** 
+/**
  * snmp network interfaces
  *   Url reference: http://nagios.manubulon.com/snmp_int.html
  */
 
 object CheckCommand "snmp-interface" {
 	import "snmp-manubulon-command"
 
-	command = [ ManubulonPluginDir + "/check_snmp_int.pl" ]
+	command = [ "perl", ManubulonPluginDir + "/check_snmp_int.pl" ]
```

Comment:
As explained in #5128 we won't add such a workaround for syncing scripts via **config sync**.